DB/src/main/java/io/deephaven/db/v2/select/AbstractConditionFilter.java

```diff
@@ -177,6 +209,7 @@ public Index filter(Index selection, Index fullSet, Table table, boolean usePrev
     }
 
     protected abstract Filter getFilter(Table table, Index fullSet) throws InstantiationException, IllegalAccessException, NoSuchMethodException, InvocationTargetException;
+    protected abstract void setFilter(Filter filter);
```

Comment:
I would add a JavaDoc to this function, explaining why it exists. I'm confident that this pattern you've introduced (allowing explicitly setting the filter, rather than waiting for code generation) will work fine, but I'm a little ambivalent about adding secondary code paths like this. Maybe we should add another layer to the class hierarchy to make this more "standard", e.g. introduce an abstract method `initializeFilter` and implement `getFilter` at that level.

Reply:
I feel the same way. I don't much like leaving behind extra state like this. If there were another way to do it, I would prefer that.

Also, when there is a filter set, all future calls to `getFilter(Table table, Index fullSet)` will ignore the `table` and `fullSet` variables. I do not understand the code well enough to know whether this is the right thing.